src/common/src/platform_posix_dir.cpp

```diff
@@ -46,6 +65,24 @@ isDirectory(const std::string &path)
    return S_ISDIR(info.st_mode);
 }
 
+std::string
+makeConfigPath(const std::string &filename)
```

Comment:
The common project is meant to contain things which are not specific to decaf at all.  You should move this up to the emulator instead, and simply provide a method to retrieve the users HOME.